### ISSUE TITLE
Build problems

### ISSUE THREAD

**Issue by bk138** [jfeinstein10/SlidingMenu#1]
Hi, do we have to use your patched version of ActionBarSherlock? It doesn't work with the upstream sources...

**Comment by jfeinstein10** [comment]
Upstream sources meaning? And what exactly wasn't working?
Right now you can use regular ActionBarSherlock, but you'll have to maintain everything at the Activity level yourself. With a few more details about the problems you're having I may be able to fix it for you :)

**Comment by bk138** [comment]
Upstream == actionbarsherlock upstream == https://github.com/JakeWharton/ActionBarSherlock.

For instance, SlidingMenu uses  `((ActionBarSherlockCompat)getSherlock()).installDecor(mainView);` but upstream just has `.installDecor()`, that's why I meant "patched version of actionbarsherlock". I kinda gotit working now with your actionbarsherlock stuff, thanks!

**Comment by developernotes** [comment]
I am running into the same problem.  I am using maven to manage my dependencies and there four versions of ActionBarSherlock available (i.e., 4.0.0, 4.0.1, 4.0.2 and 4.1.0) and I see the same issue in all versions.  The call to `((ActionBarSherlockCompat)getSherlock()).installDecor(mainView);` fails because `installDecor` is private and it no longer accepts an argument.  Is there an alternative call that can be made for this to work?

**Comment by jfeinstein10** [comment]
Yeah this method call (`((ActionBarSherlockCompat)getSherlock()).installDecor(mainView);`) is the entire reason I had to patch ActionBarSherlock. What I wanted to do while patching ActionBarSherlock is to house the ActionBar in the above view of the SlidingMenu. This would be done in the window decor usually, so I had to expose some methods that aren't usually exposed in order to install all of this into some arbitrary view. I spent about a week staring at this problem and I didn't see any other calls that could make this work, so I made my own and included the patched version.
The only reason somebody would use the Activities that I provide is if they want that effect (having the ActionBar slide). If you don't want your ActionBar to slide, then I would advise against using my Activities for now and managing the SlidingMenu as you would any other View. I'll soon go in and make regular Activities that cook in the SlidingMenu functionality.
Sorry for all the confusion :(. I'll try to make this more clear in the ReadMe / Wiki.
Does this explain everything?
